src/site/content/en/blog/file-handling/index.md

```diff
@@ -178,22 +182,38 @@ out the [implementation][demo-source] in the source code.
   </figcaption>
 </figure>
 
-## Security and permissions
+## Security
 
 The Chrome team has designed and implemented the File Handling API using the core principles defined
 in [Controlling Access to Powerful Web Platform Features][powerful-apis], including user control,
 transparency, and ergonomics.
 
+## Permissions, permissions persistence, and file handler updates
+
+To ensure user trust and the safety of users' files when the File Handling API is used to open a file,
+a permission prompt will be shown before a PWA can view a file. This permission prompt will be shown
+right after either of the following scenarios:
+
+1. The user selects the PWA to open a file, so that the permission is tightly coupled to the action of
+   opening a file using the PWA, making it more understandable and relevant.
+1. The site loads without the file, so that the user has an expectation of what the PWA is and why it
+   would like to view the file.
```

Comment:
```suggestion
* The user selects the PWA to open a file, so that the permission is tightly coupled to the action of
   opening a file using the PWA, making it more understandable and relevant.
* The site loads without the file, so that the user has an expectation of what the PWA is and why it
   would like to view the file.
```